example/__tests__/person.test.js

```diff
@@ -0,0 +1,26 @@
+import { graphql } from 'graphql';
+import schema from '../src/schema';
+
+const query = `
+  query persons {
+    persons {
+      __typename
+      uid
+      personId
+      name
+    }
+  }
+`;
+
+describe('Person query', () => {
+  it('Check recived object shape', async () => {
```

Comment:
received 😉

Reply:
Yea, also "Check" should be lowercase. We're happy to merge PRs with such changes! :)

Reply:
Also I'd delete the `describe` block. It's usually unnecessary complication in tests.